### PR TITLE
Reduce sponsor logo size on Global Thought Leaders page for WTO

### PR DESCRIPTION
**Pull request by Alixers505**
https://southcomm.atlassian.net/browse/DEV-276

New:
Screen Shot 2020-10-20 at 5 31 23 PM
Screen Shot 2020-10-20 at 5 28 23 PM

### CODE REVIEW

File: sites/watertechonline.com/server/styles/index.scss [hunk 1]
```diff
@@ -24,3 +24,20 @@ $theme-site-header-breakpoints: (
 
 @import "../../node_modules/@endeavor-business-media/package-shared/scss/core";
 @import "../../node_modules/@endeavor-business-media/package-common/scss/algolia";
+
+// Custom styling for global-thought-leaders website section
+
+/* stylelint-disable */
```
Comment (B77Mills) [inline sites/watertechonline.com/server/styles/index.scss:30]: @Alixers505  We shouldn't disable lint to get this done.  With the change from class to modifier in the section page override the correct version should look something like this

```SCSS
.node {
  $self: &;
  &--website-section-35711-sponsor {
    #{ $self } {
      &__image-wrapper {
        height: auto;
        min-height: auto;
        margin: 0 auto;
      }
    }
  }
}
```

**Review by B77Mills — CHANGES_REQUESTED** [review]
@Alixers505 Let me know if you have any questions.

File: sites/watertechonline.com/server/styles/index.scss (hunk 1 above)
Comment (Alixers505) [reply sites/watertechonline.com/server/styles/index.scss:30]: The height and min-height settings are manually set within the component to display inline in the html. So it requires "!important" to override it. Important isn't allowed by the linter settings.

Comment (B77Mills) [reply sites/watertechonline.com/server/styles/index.scss:30]: @Alixers505  to that point those are really un needed.  The height is a calculated value based off the width.  All you should need to achieve this look is 
```SCSS
.node {
  $self: &;
  &--website-section-35711-sponsor {
    #{ $self } {
      &__image-wrapper {
        margin: 0 auto;
      }
    }
  }
}
```

Comment (Alixers505) [reply sites/watertechonline.com/server/styles/index.scss:30]: *min-width, not min-height

Comment (Alixers505) [reply sites/watertechonline.com/server/styles/index.scss:30]: @B77Mills - that does solve it for desktop, yes, but not for mobile phones. You can see it drastically offset on older phones and slightly offset on newer phones:
Screen Shot 2020-10-23 at 9 30 09 AM
Screen Shot 2020-10-23 at 9 31 05 AM
Screen Shot 2020-10-23 at 9 30 53 AM
Screen Shot 2020-10-23 at 9 31 57 AM

Comment (B77Mills) [reply sites/watertechonline.com/server/styles/index.scss:30]: If the important thing needs to stay we should still only disable lint on the single lines 
```scss
.node {
  $self: &;
  &--website-section-35711-sponsor {
    #{ $self } {
      &__image-wrapper,
      &__image {
        min-width: auto !important; /* stylelint-disable-line */
        height: auto !important; /* stylelint-disable-line */
        margin: 0 auto;
      }
    }
  }
}
```

This with the other change to the file should work with a more narrowed down selector for the css override.

**Comment by Alixers505** [comment]
This has been up for a while, so I'm starting to get errors when I try to push up new changes. Hence the long list of commit changes.

**Comment by Alixers505** [comment]
I'm going to close this pull request and open a new one. Not sure why it didn't rebase correctly in this branch, and I'd rather not worry about it.

**Pull request closed without merging**